docker/Dockerfile.aarch64_cross_compile

```diff
@@ -0,0 +1,71 @@
+FROM centos:7.6.1810
+
+# Install requirements
+RUN  set -x && \
+  yum -y install epel-release && \
+  yum -y install wget tar git make autoconf automake libtool openssl-devel ninja-build gcc-c++
+
+# Install Java and Golang
+RUN yum install -y java-1.8.0-openjdk-devel golang
+ENV JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk/"
```

Comment:
@wangxiyuan can we apply the same fixes from https://github.com/netty/netty/pull/9804 ? namly extracting gcc version, using mvnw instead of installing from some http site.